src/Core/Prefab/GlobeView.js

```diff
@@ -81,28 +81,15 @@ class GlobeView extends View {
      *
      * @param {HTMLDivElement} viewerDiv - Where to attach the view and display it
      * in the DOM.
-     * @param {object|Coordinates} coordCarto - An object containing three
-     * properties: longitude, latitude and altitude. It will help placing the camera
-     * on the globe at the creation.
+     * @param {CameraTransformOptions} placement - An object to place view
      * @param {object=} options - See options of {@link View}.
      */
-    constructor(viewerDiv, coordCarto, options = {}) {
+    constructor(viewerDiv, placement, options = {}) {
```

Comment:
You need to have `placement = {}` or it will break if it is `undefined` and `options.noControls = true`.

Reply:
done